catalog-rest-service/src/main/resources/ui/src/utils/EntityVersionUtils.tsx

```diff
@@ -0,0 +1,196 @@
+import classNames from 'classnames';
+import { diffArrays, diffWordsWithSpace } from 'diff';
+import { isUndefined } from 'lodash';
+import React, { Fragment } from 'react';
+import ReactDOMServer from 'react-dom/server';
+import ReactMarkdown from 'react-markdown';
+import rehypeRaw from 'rehype-raw';
+import gfm from 'remark-gfm';
+import {
+  ChangeDescription,
+  FieldChange,
+} from '../generated/entity/services/databaseService';
+import { TagLabel } from '../generated/type/tagLabel';
+import { isValidJSONString } from './StringsUtils';
+
+/*eslint-disable */
+const parseMarkdown = (
+  content: string,
+  className: string,
+  isNewLine: boolean
+) => {
+  return (
+    <ReactMarkdown
+      children={content
+        .replaceAll(/&lt;/g, '<')
+        .replaceAll(/&gt;/g, '>')
+        .replaceAll('\\', '')}
+      components={{
+        h1: 'p',
+        h2: 'p',
+        h3: 'p',
+        h4: 'p',
+        h5: 'p',
+        h6: 'p',
+        p: ({ node, children, ...props }) => {
+          return (
+            <>
+              {isNewLine ? (
+                <p className={className} {...props}>
+                  {children}
+                </p>
+              ) : (
+                <span className={className} {...props}>
+                  {children}
+                </span>
+              )}
+            </>
+          );
+        },
+        ul: ({ node, children, ...props }) => {
+          const { ordered: _ordered, ...rest } = props;
+          return (
+            <ul className={className} style={{ marginLeft: '16px' }} {...rest}>
+              {children}
+            </ul>
+          );
+        },
+      }}
+      remarkPlugins={[gfm]}
+      rehypePlugins={[[rehypeRaw, { allowDangerousHtml: false }]]}
+    />
+  );
+};
+/*eslint-disable */
```

Comment:
same here, and no need to disable it twice

Reply:
sure will remove it.